<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pbLoad">
      <property name="text">
       <string>Ucitaj gradove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Gradovi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lvCities"/>
    </item>
    <item>
     <widget class="QPushButton" name="pbBuild">
      <property name="text">
       <string>Izgraditi puteve</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Putevi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lvRoads"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
